main.rb

```diff
@@ -0,0 +1,46 @@
+require './app'
+
+def main
+  books = []
+  people = []
+  rentals = []
+
+  selection(books, people, rentals)
+end
+
+def main_menu
+  puts 'Welcome to School library App!'
+  puts 'Please choose an option by entering a number:
+  1 - List all books
+  2 - List all people
+  3 - Create a person
+  4 - Create a book
+  5 - Create a rental
+  6 - List all rentals for a given person id
+  7 - Exit'
```

Comment:
- [ ] [OPTIONAL] Kindly make the `Exit` work as now the program continues when I press this case.
![image](https://user-images.githubusercontent.com/24703171/182944386-3147de73-1bd6-4f19-af19-148f49ba9d81.png)